core/controllers/dashboard.py

```diff
@@ -190,8 +195,33 @@ def _get_intro_card_color(category):
             key=lambda x: (x['num_open_threads'], x['last_updated']),
             reverse=True)
 
+        for collection_summary in subscribed_collection_summaries:
+            if collection_summary is None:
+                continue
+
+            collections_list.append({
```

Comment:
This looks very similar to a summary dict as provided by summary_services. Can we maybe make summary_service's ``_get_displayable_collection_summary_dicts`` and ``_get_displayable_exp_summary_dicts`` public and use them here for collections and explorations?